addons/l10n_de/models/res_company.py

```diff
@@ -9,3 +9,4 @@ class ResCompany(models.Model):
 
     l10n_de_stnr = fields.Char(string="St.-Nr.", help="Steuernummer. Scheme: ??FF0BBBUUUUP, e.g.: 2893081508152 https://de.wikipedia.org/wiki/Steuernummer")
     l10n_de_widnr = fields.Char(string="W-IdNr.", help="Wirtschafts-Identifikationsnummer.")
+    l10n_de_ceo_id = fields.Many2one("res.users", string="CEO", help="Geschäftsführer")
```

Comment:
use named parameters for all
also, I think `res.partner` might be better than `res.users`
also, we now use english terms for all strings